src/features/smartSearch/components/sankeyDiagram/SmartSearchSankeyEntrySegment.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import SmartSearchSankeySegment from './SmartSearchSankeySegment';
+import { useSankey } from './SmartSearchSankeyProvider';
+
+type SmartSearchSankeyEntrySegmentProps = {
+  hovered: boolean;
+};
+
+const SmartSearchSankeyEntrySegment: FC<SmartSearchSankeyEntrySegmentProps> = ({
+  hovered,
+}) => {
+  const { config, entrySegment } = useSankey();
+
+  return entrySegment ? (
+    <SmartSearchSankeySegment
+      config={{
+        ...config,
+        color: hovered ? '#bbbbbb' : config.color,
```

Comment:
I think we should use a theme color here, it will make it easier and neater.

Reply:
You're right!